File: PHP/PHP Basics/05.Function, Objects and Classes/Person.php
<?php

class Person{
    private $name;
    private $age;
    
    public function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age; 
    }
    
    public function getName(){
        return $this->name;
    }

    public function setName($value){
        $this->name = $value;
    }
    
    public function getAge(){
        return $this->age;
    }
    public function setAge($value){
        $this->age = $value;
    }
}

$pesho = new Person("Pesho", 19);

echo $pesho->getName()."<br>";
echo $pesho->getAge()."<br>";
?>

File: C# Advanced/C# Advanced/22.Exams/Demo Exam - 17 Feb 2019/03. Repository_Скелет/Repository/Person.cs

using System;

namespace Repository
{
    public class Person
    {
        public Person(string name, int age, DateTime birthday)
        {
            this.Name = name;
            this.Age = age;
            this.Birthdate = birthday;
        }

        public string Name { get; set; }

        public int Age { get; set; }

        public DateTime Birthdate { get; set; }
    }
}
